Source/WebCore/html/HTMLMediaElement.h

```diff
@@ -495,6 +495,7 @@ class HTMLMediaElement
     WEBCORE_EXPORT static HashSet<SecurityOriginData> originsInMediaCache(const String&);
     WEBCORE_EXPORT static void clearMediaCache(const String&, WallTime modifiedSince = { });
     WEBCORE_EXPORT static void clearMediaCacheForOrigins(const String&, const HashSet<SecurityOriginData>&);
+
     static void resetMediaEngines();
```

Comment:
Not needed

Reply:
I think it is, the comment above imply that those methods are grouped into a category. resetMediaEngines isn't part of it.